### PR TITLE
DOCS-15667 Updates to view & analyze docs page

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [DOCS-15667](https://jira.mongodb.org/browse/DOCSP-26301)
- [Stage](https://docs-mongodbcom-staging.corp.mongodb.com/landing/docsworker-xlarge/DOCS-15667/view-analyze/)
- [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6363cdf37f102f5cea6bc75b)

### CODE REVIEW

File: source/view-analyze.txt [hunk 1]
```diff
@@ -54,34 +65,52 @@ and data science tools.
...
       Build fast, relevant, full-text search capabilities on top of your data in
       the cloud.
 
+   .. card::
+      :headline: Atlas Data Lake
+      :url: https://www.mongodb.com/docs/datalake/
+      :icon: /images/icons/data-lake.svg
+      :icon-alt: Atlas Data Lake LogoMark
+
+      Utilize Data Lake to create analytic optimized copies of your
```
Comment (jwilliams-mongo) [inline source/view-analyze.txt:99]:
```suggestion
      Use Data Lake to create analytic optimized copies of
```

File: source/view-analyze.txt [hunk 2]
```diff
@@ -54,34 +65,52 @@ and data science tools.
...
+      Atlas Cluster data with complete workload isolation.
+
+   .. card::
+      :headline: Atlas SQL
+      :url: https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/
+      :icon: /images/icons/atlas-sql.svg
+      :icon-alt: Atlas SQL LogoMark
+
+      Create and run SQL queries to visualize, graph, and report on your
```
Comment (jwilliams-mongo) [inline source/view-analyze.txt:108]: suggestion: can we trim this somehow? it's length is causing extra whitespace to appear in the card next to it.

File: source/view-analyze.txt [hunk 3]
```diff
@@ -54,34 +65,52 @@ and data science tools.
...
       the cloud.
 
+   .. card::
+      :headline: Atlas Data Lake
+      :url: https://www.mongodb.com/docs/datalake/
+      :icon: /images/icons/data-lake.svg
+      :icon-alt: Atlas Data Lake LogoMark
+
+      Utilize Data Lake to create analytic optimized copies of your
+      Atlas Cluster data with complete workload isolation.
```
Comment (jwilliams-mongo) [inline source/view-analyze.txt:100]:
```suggestion
      your data with complete workload isolation.
```

File: source/view-analyze.txt [hunk 4]
```diff
@@ -54,34 +65,52 @@ and data science tools.
...
+
+   .. card::
+      :headline: Atlas SQL
+      :url: https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/
+      :icon: /images/icons/atlas-sql.svg
+      :icon-alt: Atlas SQL LogoMark
+
+      Create and run SQL queries to visualize, graph, and report on your
+      Atlas data using relational business intelligence tools.
```
Comment (jwilliams-mongo) [inline source/view-analyze.txt:109]:
```suggestion
      data using relational business intelligence tools.
```

**Review by jwilliams-mongo — APPROVED** [review]

**Review by bafMdb — APPROVED** [review]

**Pull request merged**